internal/action/show.go

```diff
@@ -158,7 +158,7 @@ func (s *Action) showHandleOutput(ctx context.Context, name string, sec gopass.S
 		return nil
 	}
 
-	ctx = out.WithNewline(ctx, ctxutil.IsTerminal(ctx))
+	ctx = out.WithNewline(ctx, false && ctxutil.IsTerminal(ctx))
```

Comment:
Why are you always disabling the newline output?
If that's correct you could remove the second part of the condition.

But I guess this is not intentional?

Reply:
Ups, leftover from debugging!